LDAR_Sim/src/simulation/simulation_manager.py

```diff
@@ -0,0 +1,298 @@
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        simulation_manager.py
+# Purpose:     Class to manage the simulation process and state
+#
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+
+import gc
+import multiprocessing as mp
+import os
+import shutil
+import sys
+from datetime import date
+from pathlib import Path
+from typing import Tuple
+
+import pandas as pd
```

Comment:
Nit: move pandas import up and separate from ldar-sim specific imports, and alphabetize the above imports

Reply:
I ran the formatter organize imports here this is how they are automatically organized. I propose we use that functionality to organize imports from now on for consistency

Reply:
Can you add that work practice to the readme as a separate PR? thanks!

Reply:
Yes